src/importexport/musicxml/internal/musicxml/export/exportmusicxml.cpp

```diff
@@ -3270,12 +3270,11 @@ static String symIdToOrnam(const SymId sid)
 
     default:
         // use other-ornament
-        const AsciiStringView name = SymNames::nameForSymId(sid);
-        return String(u"other-ornament smufl=\"%1\"").arg(String::fromAscii(name.ascii()));
         break;
     }
 
-    return String();
+    const AsciiStringView name = SymNames::nameForSymId(sid);
+    return String(u"other-ornament smufl=\"%1\"").arg(String::fromAscii(name.ascii()));
```

Comment:
ok, but why not just remove the single `return String();`?

Reply:
I'll try. Later, tomorrow.

Reply:
I seem to recall that some compilers then complain that not all possible paths through that method return a value. (They are wrong, but that doesn't help us.)

Reply:
My recollection too, but let's try...

Reply:
Seems to work, but revealed 2 more compiler errors, one on Linux and another on macOS